.circleci/config.yml

```diff
@@ -12,13 +19,37 @@ jobs:
       - run:
           name: Install dependencies
           command: |
+            sudo apt-get update
+            sudo apt-get install cmake
             gem update --system
             gem install bundler
-            bundle install --path vendor/bundle
+            bundle install --path vendor/bundle --with development test
       - run:
           name: Running checks
           command: |
             git config --global user.email "circle@example.com"
             git config --global user.name "Circle CI"
             bundle exec overcommit -s
             bundle exec overcommit -r
+  test:
+    docker:
+      - image: circleci/ruby:2.5.5-stretch-node
+    executor: ruby/default
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install cmake
+            gem update --system
+            gem install bundler
+            bundle install --path vendor/bundle --with development test
+      - run:
+          name: Running tests
+          command: |
+            bundle exec rspec
+      - run:
+          name: Check coverage
+          command: |
+            bundle exec undercover --compare origin/master
```

Comment:
Wich place you save coverage artifacts?
<img width="1147" alt="Screenshot 2021-06-04 at 2 21 21 PM" src="https://user-images.githubusercontent.com/14002692/120793889-24bee080-c540-11eb-9e04-ce2b518e9ddc.png">